packages/animations/lib/src/open_container.dart

```diff
@@ -809,13 +789,13 @@ class _OpenContainerRoute<T> extends ModalRoute<T> {
                             fit: BoxFit.fitWidth,
                             alignment: Alignment.topLeft,
                             child: SizedBox(
-                              width: _rectTween.begin.width,
-                              height: _rectTween.begin.height,
+                              width: _rectTween.begin!.width,
```

Comment:
nit: Can the begin variable be null. Maybe it makes sense to make begin end NNBD that way we don't have to use the ! all the time.

Reply:
`RectTween` is defined by the flutter framework as follows, so I can't really change it.

```dart
/// An interpolation between two rectangles.
///
/// This class specializes the interpolation of [Tween<Rect>] to use
/// [Rect.lerp].
///
/// The values can be null, representing a zero-sized rectangle at the
/// origin ([Rect.zero]).
///
/// See [Tween] for a discussion on how to use interpolation objects.
class RectTween extends Tween<Rect?> {
  /// Creates a [Rect] tween.
  ///
  /// The [begin] and [end] properties may be null; the null value
  /// is treated as an empty rect at the top left corner.
  RectTween({ Rect? begin, Rect? end }) : super(begin: begin, end: end);

  /// Returns the value this variable has at the given animation clock value.
  @override
  Rect? lerp(double t) => Rect.lerp(begin, end, t);
}
```